examples/02_meta-analyses/generate_data.py

```diff
@@ -0,0 +1,173 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 et:
+"""
+.. _metas6:
+
+========================================================
+ Simulate data for coordinate based meta-analysis (CBMA)
+========================================================
+
+Simulating data before you run your meta-analysis is a great
+way to test your assumptions and see how the meta-analysis
+would perform with simplified data
+"""
+import nimare
+from nimare.generate import create_coordinate_dataset
+from nilearn.plotting import plot_stat_map
+import numpy as np
+
+
+###############################################################################
+# Create function to perform a meta-analysis and plot results
+# -----------------------------------------------------------
+
+
+def analyze_and_plot(dset, ground_truth_foci):
+    mkda = nimare.meta.mkda.MKDADensity(kernel__r=10)
+    mkda.fit(dset)
+    corr = nimare.correct.FWECorrector(method="montecarlo", n_iters=10, n_cores=1)
+    cres = corr.transform(mkda.results)
+
+    # get the z coordinates
+    cut_coords = [c[2] for c in ground_truth_foci]
+    display = plot_stat_map(
+        cres.get_map("logp_level-voxel_corr-FWE_method-montecarlo"),
+        display_mode="z",
+        cut_coords=cut_coords,
+        draw_cross=False,
+        cmap="RdBu_r",
+    )
+    display.add_markers(ground_truth_foci)
+
+    return display
+
+
+###############################################################################
+# Create Dataset
+# --------------------------------------------------
+# In this example, each of the 30 fake studies being generated
+# select 4 coordinates from a probability map representing the probability
+# that particular coordinate will be chosen.
+# There are 4 "hot" spots centered on 3D gaussian distributions.
+# Each study has a mean sample_size of 30
```

Comment:
I'm not confident in the language I'm using here describing how the simulated data are created, suggestions for rephrase or what seems unclear?

Reply:
I think it's fairly clear. We can improve later, but I don't think it should be a blocker for this PR.